### DISCUSSION TITLE
Reporting Conda as [Information] in run log

### DISCUSSION THREAD

**Discussion by BadgerOnABike** [BurnP3/BurnP3Plus#22] [Ideas]
I know we are looking for folks to use Conda as it ensures they are in a sanitary environment, but is reporting it as information the best way?

As I prepare my runs for our course I'm seeing everything popping up as "info" even though they all ran fine. I'll test with both Conda and locally of course, but my local runs look like they may have an issue when in reality the only piece of information is that I could have run this in Conda. Not sure how I feel about that, I'd prefer it shows a success and has the reminder in the run log, maybe at the top and bottom. It also pushes that recommendation at every pipeline step. 

Thoughts @katieb1 @cstock69 @cptcitrus @LeonardoFrid @shreeramsenthi

**Comment by katieb1** [comment]
@BadgerOnABike I agree that it's a bit over the top. I think I can prevent it from having the "info" status and instead just be text in the run log, but it shows up for every pipeline stage because conda environments in SyncroSim can be "stage-specific". So SyncroSim is checking every pipeline stage and reporting back. Maybe the message also needs to be shorter and should just point to the online documentation for how to set up conda environments.

**Reply by katieb1** [reply]
I made the change and SyncroSim v2.5.7 will have the updated run log messages.

**Comment by BadgerOnABike** [comment]
Appreciate it, thank you Katie!
